repl/src/test/scala/com/cloudera/livy/repl/Python3InterpreterSpec.scala

```diff
@@ -208,15 +208,5 @@ class Python3InterpreterSpec extends BaseInterpreterSpec {
     ))
   }
 
-  // Scalastyle is treating unicode escape as non ascii characters. Turn off the check.
-  // scalastyle:off non.ascii.character.disallowed
-  it should "print unicode correctly" in withInterpreter { intp =>
-    intp.execute("print(u\"\u263A\")") should equal(Interpreter.ExecuteSuccess(
```

Comment:
@tc0312 @vanzin 
Does above testcase need to Python3? 
Does ` print("\xE2\x98\xBA")` result need to be ""\u263A""?
a  = "'\\xE2\\x98\\xBA'"
python2 
```
print(isinstance(a, bytes))
True
```

python3
```
print(isinstance(a, bytes))
False
```